mibitracker/request_helpers.py

```diff
@@ -241,9 +241,9 @@ def copy_run(self, old_label, new_label, **kwargs):
         run_data = {
             'instrument': data['instrument']['id'],  # required
             'slides': ','.join(str(d['id']) for d in data['slide_ids']),
-            'aperture': data['aperture']['id'],  # required
             'label': new_label,
-            'magnification': data['magnification'],  # required, FOV size
+            'fov_size': data['fov_size'],
```

Comment:
`'fov_size'` gets ignored here, here we also retained the ability to post with `'magnification'` for backwards compatibility.

Reply:
Sorry, I was mistaken, either `fov_size` or `magnification` works as the parameter in the request, so your change is right,